packages/devtools_server/lib/src/memory_profile.dart

```diff
@@ -9,7 +9,7 @@ import 'package:devtools_shared/devtools_shared.dart';
 import 'package:intl/intl.dart' as intl;
 import 'package:vm_service/vm_service.dart';
 
-import 'service_registrations.dart' as registrations;
```

Comment:
I don't think this file rename makes sense, as not all of the service code in service_registrations.dart are pertaining to memory features

Reply:
OK, I reverted the rename.